### ISSUE TITLE
azure-client-authentication using msal4j

### ISSUE THREAD

**Issue by apescione** [Azure/autorest-clientruntime-for-java#693]
In our application, we use azure-client-authentication that leverages on adal4j. In order to migrate towards the new suggested library by Microsoft, my question is if and when this library will be upgraded to a new one.

Thank you in Advance

**Comment by weidongxu-microsoft** [comment]
@apescione 

Please check if your usage can be provided by these SDKs https://azure.github.io/azure-sdk/releases/latest/java.html
They uses the new azure-identity lib, which depends on msal4j lib.

**Comment by apescione** [comment]
Yes, great, I need to perform a little bit of refactoring but this library sounds good. Thank you
